porting/installing-16-04.rst

```diff
@@ -5,76 +5,60 @@ Installing Ubuntu Touch 16.04 images on Halium
 
     These steps  will wipe **all** of the data on your device. If there is anything that you would like to keep, ensure it is backed up and copied off of the device before continuing.
 
-Now that you've :doc:`built ubports-boot <building-ubports-boot>`, we'll use a script called ``rootstock-touch-install`` to install an Ubuntu Touch rootfs on your device.
+Now that you've :doc:`built halium-boot <building-halium-boot>`, we'll use a script called ``halium-install`` to install an Ubuntu Touch rootfs on your device.
 
```

Comment:
-> Now that ..., we're ready to install Ubuntu Touch on your device.

we're not literally using a script of that name right now, and there is no need to mention the name here anyway, and we're not ONLY ready to install rootfs, but we're installing the whole thing inc boot